qiita_db/study.py

```diff
@@ -487,14 +487,44 @@ def title(self, title):
         Parameters
         ----------
         title : str
-            The new study title
+            The study title
         """
         with qdb.sql_connection.TRN:
             sql = """UPDATE qiita.{0} SET study_title = %s
                      WHERE study_id = %s""".format(self._table)
             qdb.sql_connection.TRN.add(sql, [title, self._id])
             return qdb.sql_connection.TRN.execute()
 
+    @property
+    def public_raw_download(self):
+        """Returns the public_raw_download of the study
+
+        Returns
+        -------
+        str
+            public_raw_download of study
+        """
+        with qdb.sql_connection.TRN:
+            sql = """SELECT public_raw_download FROM qiita.{0}
+                     WHERE study_id = %s""".format(self._table)
+            qdb.sql_connection.TRN.add(sql, [self._id])
+            return qdb.sql_connection.TRN.execute_fetchlast()
+
+    @public_raw_download.setter
+    def public_raw_download(self, public_raw_download):
+        """Sets the public_raw_download of the study
+
+        Parameters
+        ----------
+        public_raw_download : bool
+            The study public_raw_download
+        """
```

Comment:
Returns is missing in documentation

Reply:
This is a setter ... doesn't have a return

Reply:
d'oh!